sys-apps/fwupd/metadata.xml

```diff
@@ -1,24 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <maintainer type="person">
-    <email>polynomial-c@gentoo.org</email>
-    <name>Lars Wendler</name>
-  </maintainer>
-  <use>
-    <flag name="agent">Enable update notification agent</flag>
-    <flag name="colorhug">Enable ColorHug support through<pkg>x11-misc/colord</pkg> package</flag>
-    <flag name="consolekit">Use <pkg>sys-auth/consolekit</pkg> for session tracking</flag>
-    <flag name="dell">Enable Dell-specific support.</flag>
-    <flag name="gpg">Enable GPG support.</flag>
-    <flag name="man">Build and install man pages</flag>
-    <flag name="nvme">Build and install nvme plugin</flag>
-    <flag name="pkcs7">Enable PKCS7 verification support via <pkg>net-libs/gnutls</pkg></flag>
-    <flag name="redfish">Install redfish plugin</flag>
-    <flag name="thunderbolt">Install thunderbolt plugin</flag>
-    <flag name="uefi">Enable UEFI support</flag>
-  </use>
-  <upstream>
-    <remote-id type="github">hughsie/fwupd</remote-id>
-  </upstream>
+	<maintainer type="person">
+		<email>polynomial-c@gentoo.org</email>
+		<name>Lars Wendler</name>
+	</maintainer>
+	<use>
+		<flag name="agent">Enable update notification agent</flag>
```

Comment:
Styling here as well :)

Good job otherwise, the meson_use is so much cleaner.

Reply:
<3